oat-sa/tao-core
# accessibility-issues-on-delivery-page

https://oat-sa.atlassian.net/browse/ACTP-362

Accessibility issues on Delivery page when there are tests 'In progress'

**How to test:**
- login to ACT as a Guest or as user
- check focus visibility on `logout` button
- check is deliveries main tag is &lt;main&gt;
- check landmarks of `banner` and `navigation` is not mixed

**related to PR's:**
https://github.com/oat-sa/extension-tao-delivery/pull/440
